client/src/app/+about/about.component.html

```diff
@@ -1,3 +1,13 @@
-<p>
-  about works!
-</p>
+
+    <h3> אודות</h3>
+    <p>נכון לסוף 2013, כטריליון שקלים מנוהלים בקרנות הפנסיה וקופות הגמל בישראל.</p>כספים אלו - המיועדים להבטחת רמת חיינו לאחר גיל הפרישה לפנסיה - מנוהלים ע"י הגופים המוסדיים בישראל
+    ומושקעים מדי יום בשוקי ההון והכספים בארץ ובחו"ל.
+    <p>פרויקט פנסיה פתוחה, מבית <a href="http://www.hasadna.org.il/">הסדנא לידע ציבורי</a>, שם לו למטרה לחשוף את הדרך שבה כספי הפנסיה שלנו מנוהלים וכיצד הם מושקעים. באמצעות פנסיה פתוחה, תוכלו לגלות גם איך מושקעים כספי הפנסיה של הציבור כולו, וגם איך מושקעים כספי הפנסיה בקופה ספציפית (למשל, הקופה שלכם). תוכלו לענות על שאלות כמו: </p>
+    <ul>
+      <li>איזה אחוז מכספי הפנסיה שלנו מושקע בבנקים?</li>
+      <li>כמה כסף מלוות קופות הפנסיה לחברות הנדל"ן?</li>
+      <li>איזה חלק מהפנסיה שלנו מושקע בחו"ל?</li>
+    </ul>
+    <p>הנתונים שמוצגים בפנסיה פתוחה לקוחים מהדו”חות המפורסמים ע"י הגופים המנהלים של קרנות הפנסיה. דו”חות אלו המציגים פירוט ברמת הנכס הבודד של הנכסים המוחזקים בכל קרן. הדו”חות מתפרסמים בסוף כל רבעון, ופנסיה פתוחה מעדכנת את מאגר הנתונים שלה לאחר מכן. לפנסיה פתוחה אין כל אחריות לנכונות ו/או לשלמות הנתונים המוצגים.&nbsp;</p>
+    <p>בשלב זה, פנסיה פתוחה מציגה רק את הנתונים של חברת מגדל. בעתיד הקרוב, פנסיה פתוחה תקיף את כל שוק הפנסיה: קרנות פנסיה, קופות גמל וביטוחי מנהלים.</p>
+    
```

Comment:
@eyalcohen4 Pls delete the white space in this line (keep the new line at the end of the file, but without spaces/tabs).